cmd/kyma/init/function/opts.go

```diff
@@ -37,6 +40,11 @@ func (o *Options) setDefaults(defaultNamespace string) (err error) {
 		}
 	}
 
+	if o.Name == "" {
+		rand.Seed(time.Now().UnixNano())
```

Comment:
Seeding the RNG is only necessary if you want different pseudo-random sequences in each run, which is probably not needed here.

If you still think this is necessary though, I would expect the seeding to happen inside `GenerateName`. The consumer having to do it, leaks implementation details of `GenerateName`.

Reply:
I think this line is necessary, so I moved this to implementation GenerateName

Reply:
It that case, this PR can be merged after https://github.com/kyma-incubator/hydroform/pull/217 and updating the `go.mod` file.

Reply:
Yes, of course, but I'm waiting for your review because if you will have another comments I will have to create new PR to hydroform.

Reply:
AH I see, I do not have any further comments. As I said in the previous comment, as soon as the other PR is merged and the go.mod file updated here, this can be merged.

Reply:
Done